docs/dapps/nfts/intro-to-erc721s.md

```diff
@@ -44,20 +44,40 @@ Once the image and metadata files are ready, we can prepare to deploy a smart co
 
 ## Preparing Your Environment
 
-### MetaMask Extension
+### Core Extension
 
-You'll need the MetaMask Extension installed on whatever browser you're using to
+You'll need the Core Extension installed on whatever browser you're using to
 be able to fund the deployment of the smart contract. If you've not done so
-already, download MetaMask and [add the Fuji network to
-MetaMask](../launch-your-ethereum-dapp.md#fuji-testnet-settings). Create or
-import a Fuji account as necessary.
+already, download Core and enable Testnet Mode.
+To do that, go to **Settings** and click on **Advanced**.
+
+![Settings image 1](/img/c-chain-ERC20/settings1.png)
+
+Here, turn on the **Testnet Mode** feature. This will automatically make Core switch to
+Fuji Testnet. 
+
+![Settings image 2](/img/c-chain-ERC20/settings2.png)
+
+:::info
+
+If you are using other wallets, like Core or MetaMask, you can add the Fuji 
```

Comment:
I would reword this slightly - the sentence above mentions how to toggle on the Tesnet mode in Core, however, this line could be confusing saying "wallets, like **Core** or MetaMask...".

Reply:
That's true, thank you. I will remove Core from there.